src/main/java/com/github/wovnio/wovnjava/Headers.java

```diff
@@ -20,10 +20,12 @@
 
     private HttpServletRequest request;
     private String pathLang;
+    private PatternHandler patternHandler;
 
-    Headers(HttpServletRequest r, Settings s) {
+    Headers(HttpServletRequest r, Settings s, PatternHandler patternHandler) {
```

Comment:
For clarity I would add information to the name of the type. I would prefer `PatternHandler` to be call `UrlLangPatternHandler`.

Reply:
You're right, PatternHandler` is not great. How about `UrlPatternHandler` to mirror the `urlPattern` setting?

Reply:
I never thought `urlPattern` was good enough :'(
I think "language" is an important information that should be in the name.

Reply:
Renamed PatternHandler to UrlLangPatternHandler

Reply:
May as well go ahead with UrlLanguagePatternHandler. Not a huge fan of abbreviations in class names.

Reply:
Thanks